packages/fxa-auth-server/config/index.ts

```diff
@@ -2213,6 +2213,18 @@ const convictConf = convict({
         env: 'RECOVERY_PHONE__SMS__MAX_RETRIES',
         format: Number,
       },
+      smsPumpingRiskThreshold: {
+        default: 75,
```

Comment:
I'm cleaning all these up. Thanks for the catch on the typo.

Reply:
Thanks! Spotted 6 remaining mentions of "sim" pumping (e.g., in env var name here)